lib/PrintCommon.ml

```diff
@@ -33,6 +33,8 @@ let width_to_string = function
   | Bool -> "bool"
   | SizeT -> if !Options.linux_ints then "size_t" else "size"
   | PtrdiffT -> if !Options.linux_ints then "ptrdiff_t" else "ptrdiff"
+  | Float32 -> "float32"
```

Comment:
you could use `float` and `double` here since they are guaranteed to be 32 and 64 bits respectively